api/app/Generators/CandidateProfileCsv.php

```diff
@@ -218,15 +233,26 @@ public function generate()
     private function generatePoolHeaders()
     {
 
-        Pool::with(['generalQuestions', 'poolSkills' => ['skill']])
+        Pool::with(['generalQuestions', 'screeningQuestions', 'poolSkills' => ['skill']])
             ->whereHas('poolCandidates', function ($query) {
                 $query->whereIn('id', $this->ids);
             })->chunk(100, function ($pools) {
                 foreach ($pools as $pool) {
                     if ($pool->generalQuestions->count() > 0) {
                         foreach ($pool->generalQuestions as $question) {
-                            $this->questionIds[] = $question->id;
-                            $this->generatedHeaders['general_questions'][] = $question->question[$this->lang];
+                            $this->generalQuestionIds[] = $question->id;
+                            $this->generatedHeaders['general_questions'][] = $this->lang == 'en' ?
+                            'General question: '.$question->question[$this->lang] :
+                            'Frenchy string: '.$question->question[$this->lang];
```

Comment:
What should be the string here?

General questions plural

![image](https://github.com/user-attachments/assets/4fedaeb3-9122-488a-a149-c5d32a51d23a)

![image](https://github.com/user-attachments/assets/8662eb81-5f95-48ce-a931-a0365d3e6260)

Reply:
@tristan-orourke 
Thoughts on both comments? (Not sure who to ask in specific)

Reply:
Stick with singular: "General question: {the actual question}"

Reply:
Sorry for the confusion, but yeah, I meant French 😛

Reply:
I see. The singular of this is just `Question générale`

Reply:
Okay, added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11093/commits/9b7967330169756d9ece240ec03f343fb488f8bb